specification/e-referrals-service-api.yaml

```diff
@@ -2369,7 +2380,7 @@ paths:
           - `REFERRING_CLINICIAN`
           - `REFERRING_CLINICIAN_ADMIN`
 
-        You should obtain a `Unique Booking Reference Number` (UBRN) before using this endpoint, as the current UBRN version is required
+        You should obtain a `Unique Booking Reference Number` (UBRN) before using this endpoint, as the current UBRN version is required.
```

Comment:
I wonder if Unique Booking Reference Number should be referenced using single quotes here also.

Reply:
See https://nhsd-jira.digital.nhs.uk/browse/ERSSUP-60881 and images attached.